pkg/api/norman/customization/aks/handler.go

```diff
@@ -22,6 +25,12 @@ import (
 	"github.com/sirupsen/logrus"
 )
 
+const (
+	tenantIDAnnotation          = "cluster.management.cattle.io/azure-tenant-id"
+	tenantIDTimestampAnnotation = "cluster.management.cattle.io/azure-tenant-id-created-at"
+	tenantIDTimeout             = time.Hour
```

Comment:
On the previous conversation, we could make this 24 hours. If it ever changes it would be very rare. But I don't think 1 hour really hurts either. If anything its going to be that first flurry of requests that could cause a problem.

Reply:
Fair enough. Removing the timeout isn't a terrible idea but there are downsides. Let's just leave at an hour.